### PR TITLE
CEV-25224, CEV-25225, CEV-25071

### PR DESCRIPTION
**Pull request by yevgenp**
Bump plugin version
EC-DslDeploy import/export procedures: add support for characters in object names that isn't supported in file/directory name on Windows
Parameters of EC-DslDeploy procedures accepting Boolean value should have 'checkbox' type

Preflight: https://viva.beescloud.com/commander/link/jobDetails/jobs/aa4100b5-c2b4-11ea-a4ec-42010ac90206

### CODE REVIEW

File: dsl/procedures/installProject/form.xml [hunk 1]
```diff
@@ -30,19 +22,33 @@
...
-        <type>entry</type>
+        <type>checkbox</type>
         <label>Local mode</label>
         <property>localMode</property>
         <required>0</required>
+        <checkedValue>1</checkedValue>
+        <uncheckedValue>0</uncheckedValue>
         <value>0</value>
         <documentation>Do not send files to the server with help of clientFiles argument of the evalDsl operation in local mode.</documentation>
+    </formElement>\
```
Comment (itykhan) [inline dsl/procedures/installProject/form.xml:43]: extra \
please leave 'Overwrite mode' on its old place

File: dsl/procedures/importDslFromGit/form.xml [hunk 2]
```diff
@@ -81,6 +83,26 @@
     <required>0</required>
   </formElement>
 
+  <formElement>
```
Comment (itykhan) [inline dsl/procedures/importDslFromGit/form.xml:86]: please move parameters after Clone?

File: dsl/procedures/installDslFromDirectory/form.xml [hunk 3]
```diff
@@ -49,19 +41,33 @@
...
+    <type>checkbox</type>
     <label>Local mode</label>
     <property>localMode</property>
     <required>0</required>
+    <checkedValue>1</checkedValue>
+    <uncheckedValue>0</uncheckedValue>
     <value>0</value>
     <documentation>Do not send files to the server with help of clientFiles argument of the evalDsl operation in local mode.</documentation>
   </formElement>
+  <formElement>
```
Comment (itykhan) [inline dsl/procedures/installDslFromDirectory/form.xml:63]: please move Overwrite mode to its old place

File: dsl/procedures/installProject/form.xml (hunk 1 above)
Comment (yevgenp) [reply dsl/procedures/installProject/form.xml:53]: Done

File: dsl/procedures/installProject/form.xml [hunk 4]
```diff
@@ -30,19 +32,23 @@
...
+        <type>checkbox</type>
         <label>Local mode</label>
         <property>localMode</property>
         <required>0</required>
+        <checkedValue>1</checkedValue>
+        <uncheckedValue>0</uncheckedValue>
         <value>0</value>
         <documentation>Do not send files to the server with help of clientFiles argument of the evalDsl operation in local mode.</documentation>
-    </formElement>
+    </formElement>\
```
Comment (itykhan) [inline dsl/procedures/installProject/form.xml:53]: I still see extra \ here

Comment (yevgenp) [reply dsl/procedures/installProject/form.xml]: Removed

**Pull request merged**